test/integration-orientdb/bugs/Test_Edge_Creation

```diff
@@ -0,0 +1,120 @@
+var assert = require('assert');
```

Comment:
Please rename this file to something like `116-test_edge_creation.js` so it's easier to match it to a particular issue.